bps/caching/cp_caching.yaml

```diff
@@ -0,0 +1,17 @@
+# There are recommended configs for caching data read from the S3
+# datastore used by the embargo rack at USDF.
+#
+# Use it by adding
+#
+#  includeConfigs:
+#    - ${CP_PIPE_DIR}/bps/caching/cp_caching.yaml
+#
+# (with no outer indentation) to your BPS config file.
+
+environment:
+  # Default caching config
+  # Each pipetask creates its own local cache folder,
+  # $TMPDIR/butler-cache-dir-*
+  DAF_BUTLER_CONFIG_PATH: ${CP_PIPE_DIR}/bps/caching/LSSTCam/usdf/config:${DAF_BUTLER_CONFIG_PATH}
```

Comment:
I think this needs to be 
```
 DAF_BUTLER_CONFIG_PATH: ${CP_PIPE_DIR}/bps/caching/config:${DAF_BUTLER_CONFIG_PATH}
```
or the full path containing the `fileDatastore.yaml` changed.

Reply:
Thank you for spotting that. I forgot to update this following the move of the `fileDatastore.yaml` file to its new location.